enterprise/server/remote_execution/dirtools/dirtools.go

```diff
@@ -401,15 +402,15 @@ func UploadTree(ctx context.Context, env environment.Env, dirHelper *DirHelper,
 		return uploadableFile.FileNode(), nil
 	}
 
-	var uploadDirFn func(parentDir, dirName string) (*repb.DirectoryNode, error)
-	uploadDirFn = func(parentDir, dirName string) (*repb.DirectoryNode, error) {
+	var visitDir func(parentFullPath, dirName string) (*repb.DirectoryNode, error)
```

Comment:
supernit: inline this? unless that doesn't compile?

Reply:
yeah, doesn't compile unfortunately